tests/core/test_playback.py

```diff
@@ -505,15 +548,13 @@ def test_play_when_stopped_emits_events(self, listener_mock):
         self.core.playback.play(tl_tracks[0])
         self.replay_events()
 
-        self.assertListEqual(
-            [
-                mock.call(
-                    'playback_state_changed',
-                    old_state='stopped', new_state='playing'),
-                mock.call(
-                    'track_playback_started', tl_track=tl_tracks[0]),
-            ],
-            listener_mock.send.mock_calls)
+        assert [
+            mock.call(
+                'playback_state_changed',
+                old_state='stopped', new_state='playing'),
+            mock.call(
+                'track_playback_started', tl_track=tl_tracks[0]),
+        ] == listener_mock.send.mock_calls
```

Comment:
I know that you just ported from unittest to pytest asserts. Though, I'd flip the two parts around, so it becomes `assert actual == expected` instead of `assert expected == actual`.